xonotic.patch

```diff
@@ -96,13 +97,13 @@ index 50d829e..85fbe91 100644
 +
 +.PHONY: all-zip-source
 +all-zip-source:
-+	( cd source/d0_blind_id && ./configure --enable-static --disable-shared $(RIJNDAELDETECT_CONFIGURE) )
-+	$(MAKE) -C source/d0_blind_id
++	( cd source/d0_blind_id && ./configure --prefix /app --enable-shared --enable-static $(RIJNDAELDETECT_CONFIGURE) )
```

Comment:
Same thing: Is it really required to manually specify the `/app` prefix?

Reply:
Here - yes. It tries to install it to /usr, which is read-only.

Reply:
Ah. Hmm, I thought that [this variable](https://github.com/flathub/org.xonotic.Xonotic/blob/master/org.xonotic.Xonotic.json#L30) gets applied here (since it is applied in the rest of the Makefile without issue.

Reply:
Except that it's hard-coded to `/app` (there's no other choice), I think it's resolved?

Reply:
Maybe change it to `--prefix $(PREFIX)`?

Reply:
Or the `--prefix` here is not needed anymore?

Reply:
Without `--prefix` it's still trying to install everything to `/usr`, so that's necessary unless they add support for PREFIX environment variable (or, at least, I do)